src/.eslintrc.js

```diff
@@ -23,6 +23,22 @@ module.exports = {
         ]
       }
     },
+    {
+      files: ['govuk/**/*.{cjs,js,mjs}'],
+      excludedFiles: ['govuk/**/*.test.{cjs,js,mjs}'],
```

Comment:
Which means we only need to exclude ESM tests

```suggestion
      excludedFiles: ['**/*.test.mjs'],
```